docs/quest_poi_points.md

```diff
@@ -49,6 +49,11 @@ Example quest: Secreat Communication.
 | 8318    | 3    | 6    | -6483 | -217 | 0             |
 | 8318    | 3    | 7    | -6326 | -7   | 0             |
 
+**These points are all the little corners on the blue box. idx1 represents an area defined by the points of idx2 to draw the shape.**
```

Comment:
This is already described above

> The actual points are the corners of each polygon.

Reply:
Yes I understand, but this description helps to better understand if associated with the image.